Dockerfile

```diff
@@ -3,14 +3,16 @@
 FROM python:3.9 as base
 
 WORKDIR /app
+# explicit requirements because runtime does not need ALL dependencies
+COPY requirements-pip.txt .
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip install -r requirements-pip.txt

```

Comment:
Is this a new file? Missing?

Reply:
hmm. shows in the list of files in github UI for me